ecs-cli/modules/cli/local/project/project.go

```diff
@@ -32,10 +33,11 @@ import (
 	"github.com/urfave/cli"
 )
 
-const (
-	LocalOutDefaultFileName = "./docker-compose.local.yml"
+var (
+	wd, _                   = os.Getwd()
+	LocalOutDefaultFileName = filepath.Join(wd, "docker-compose.local.yml")
```

Comment:
Can we leverage this variable in `local ps` or `local down`? https://github.com/aws/amazon-ecs-cli/blob/ecs-local/ecs-cli/modules/cli/local/ps_app.go#L89

Removing code duplication, the main concern is that for whatever reason if we rename `docker-compose.local.yml` to something else we will need to remember to update it in multiple locations instead of just here.

Reply:
1. Yeah I agree with you. I think that is doable and will have a try.

2. "flags.LocalOutputFlag" is the string flag for customized yml name. If that flag is set, the default one will not be used.